lib/open_project/files.rb

```diff
@@ -56,5 +56,14 @@ def create_uploaded_file(name: 'test.txt',
       tmp = create_temp_file name: name, content: content, binary: binary
       Rack::Multipart::UploadedFile.new tmp.path, content_type, binary
     end
+
+    def mock_uploaded_file(name: 'test.txt',
+                           content_type: 'text/plain',
+                           content: 'test content',
+                           binary: false)
+
+      tmp = create_temp_file name: name, content: content, binary: binary
+      Rack::Test::UploadedFile.new tmp.path, content_type, binary
+    end
```

Comment:
Tell this the guy writing `create_uploaded_file`...

I did not verify that the only productive occurence is valid after all ^^

But you are right, existing bad code is no excuse for producing new one ^^